src/test/kotlin/com/jetbrains/snakecharm/fixtures/SnakemakeResolveTestCase.kt

```diff
@@ -0,0 +1,134 @@
+package com.jetbrains.snakecharm.fixtures
+
+import com.intellij.openapi.util.Ref
+import com.intellij.openapi.util.text.StringUtil
+import com.intellij.openapi.vfs.LocalFileSystem
+import com.intellij.openapi.vfs.VfsUtil
+import com.intellij.openapi.vfs.ex.temp.TempFileSystem
+import com.intellij.psi.*
+import com.intellij.testFramework.TestDataFile
+import com.jetbrains.python.psi.LanguageLevel
+import com.jetbrains.python.psi.impl.PyBuiltinCache
+import com.jetbrains.snakecharm.SnakemakeTestCase
+import com.jetbrains.snakecharm.SnakemakeTestUtil.getTestDataPath
+import java.io.File
+import java.io.IOException
+
+abstract class SnakemakeResolveTestCase : SnakemakeTestCase() {
+    companion object {
+        protected const val MARKER = "<ref>"
+
+        fun <T : PsiElement> assertResolveResult(element: PsiElement?,
+                                                 aClass: Class<T>,
+                                                 name: String): T {
+            return assertResolveResult(element, aClass, name, null)
+        }
+
+        fun <T : PsiElement> assertResolveResult(element: PsiElement?,
+                                                 aClass: Class<T>,
```

Comment:
To my mind in methods like JUnit assert* it is good to explicitly specify which values are expected, which parameters are actual. So here I'd rather name variables
```
        fun <T : PsiElement> assertResolveResult(
                actualElement: PsiElement?,
                expectedClass: Class<T>,
                expectedElementName: String,
                expectedContainingFilePath: String?
        ): T {
```

E.g. similar to 

```
  public static void assertEquals(String expected, String actual) {
    Assert.assertEquals(expected, actual);
  }
```

Reply:
Done